### PR TITLE
chore: Remove superseded import and save add-ons

### PR DESCRIPTION
**Pull request by kingthorin**
Remove importLogFIles, importurls, saverawmesave, and savexmlmessage which have been superseded by exim. (Related to issue 6579)

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Review by ricekot — APPROVED** [review]

**Comment by thc202** [comment]
The file `addOns.gradle.kts` needs to be updated.

File: addOns/addOns.gradle.kts [hunk 1]
```diff
@@ -66,10 +66,7 @@ val createPullRequestNextDevIter by tasks.registering(CreatePullRequest::class)
 val releaseAddOn by tasks.registering
 
 val crowdinExcludedProjects = setOf(
-    childProjects.get("importLogFiles"),
-    childProjects.get("importurls"),
-    childProjects.get("saverawmessage"),
-    childProjects.get("savexmlmessage")
+    // None excluded currently
```
Comment (kingthorin) [inline addOns/addOns.gradle.kts:69]: Should I leave an example entry in here?
```
     childProjects.get("addonid")
```

**Comment by kingthorin** [comment]
Done

**Comment by thc202** [comment]
It needs to be kept, or remove the other code using it. It seems useful to keep it but fine if removed.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**